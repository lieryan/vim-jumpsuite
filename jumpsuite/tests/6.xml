<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
	<testsuite errors="2" failures="0" name="comments.tests.test_api.APITests-20190318162954" skipped="0" tests="2" time="0.006" timestamp="2019-03-18T16:29:54">
		<testcase classname="comments.tests.test_api.APITests" name="test_api_one" time="0.004" timestamp="2019-03-18T16:29:54">
			<error message="null value in column &quot;foobared_at&quot; violates not-null constraint
DETAIL:  Failing row contains (230, 58c74234-b218-4169-92c3-12240b300ac9, null, , , , , 2019-03-18 05:29:54.064842+00, 2019-03-18 05:29:54.065263+00, 2017-01-01 00:00:00+00, null, null, null, created, , null).
" type="IntegrityError">
<![CDATA[Traceback (most recent call last):
  File "/Users/lieryan/.virtualenvs/myenv/lib/python3.6/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
psycopg2.IntegrityError: null value in column "foobared_at" violates not-null constraint
DETAIL:  Failing row contains (230, 58c74234-b218-4169-92c3-12240b300ac9, null, , , , , 2019-03-18 05:29:54.064842+00, 2019-03-18 05:29:54.065263+00, 2017-01-01 00:00:00+00, null, null, null, created, , null).


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/user/Projects/myapp/comments/tests/test_api.py", line 35, in test_api_one
    Something.objects.create(sent_at=datetime(year=2017, month=1, day=1, tzinfo=utc)).send()
  File "/Users/lieryan/.virtualenvs/myenv/lib/python3.6/site-packages/django/db/models/manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/lieryan/.virtualenvs/myenv/lib/python3.6/site-packages/django/db/models/query.py", line 413, in create
    obj.save(force_insert=True, using=self.db)
  File "/Users/lieryan/.virtualenvs/myenv/lib/python3.6/site-packages/django/db/models/base.py", line 718, in save
    force_update=force_update, update_fields=update_fields)
  File "/Users/lieryan/.virtualenvs/myenv/lib/python3.6/site-packages/django/db/models/base.py", line 748, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "/Users/lieryan/.virtualenvs/myenv/lib/python3.6/site-packages/django/db/models/base.py", line 831, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "/Users/lieryan/.virtualenvs/myenv/lib/python3.6/site-packages/django/db/models/base.py", line 869, in _do_insert
    using=using, raw=raw)
  File "/Users/lieryan/.virtualenvs/myenv/lib/python3.6/site-packages/django/db/models/manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/lieryan/.virtualenvs/myenv/lib/python3.6/site-packages/django/db/models/query.py", line 1136, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "/Users/lieryan/.virtualenvs/myenv/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 1289, in execute_sql
    cursor.execute(sql, params)
  File "/Users/lieryan/.virtualenvs/myenv/lib/python3.6/site-packages/django/db/backends/utils.py", line 68, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/Users/lieryan/.virtualenvs/myenv/lib/python3.6/site-packages/django/db/backends/utils.py", line 77, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/Users/lieryan/.virtualenvs/myenv/lib/python3.6/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/Users/lieryan/.virtualenvs/myenv/lib/python3.6/site-packages/django/db/utils.py", line 89, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/lieryan/.virtualenvs/myenv/lib/python3.6/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
django.db.utils.IntegrityError: null value in column "foobared_at" violates not-null constraint
DETAIL:  Failing row contains (230, 58c74234-b218-4169-92c3-12240b300ac9, null, , , , , 2019-03-18 05:29:54.064842+00, 2019-03-18 05:29:54.065263+00, 2017-01-01 00:00:00+00, null, null, null, created, , null).

]]>			</error>
		</testcase>
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testsuite>
</testsuites>
