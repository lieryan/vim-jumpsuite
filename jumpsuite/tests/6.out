FAILED. Ran 2 tests in 0.006s (errors=2, failures=0, skipped=0)
/home/user/Projects/myapp/comments/tests/test_api.py:35: test_api_one : django.db.utils.IntegrityError: null value in column "foobared_at" violates not-null constraint : Something.objects.create(sent_at=datetime(year=2017, month=1, day=1, tzinfo=utc)).send()
